com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md

```diff
@@ -37,31 +27,45 @@ There are three tabs that you can use to set up your HDRP Project for different
+| **Editor Resources**           | Checks to make sure that your HDRP Asset references a [**Render Pipeline Editor Resources**](HDRP-Asset.md)  Asset.<br />Press the **Fix** button to reload the runtime resources for the HDRP Asset. |
+| **Diffusion Profile**          | Checks to make sure that your HDRP Asset references a [**Diffusion Profile**](Diffusion-Profile.md) Asset.<br />Press the **Fix** button to reload the runtime resources for the HDRP Asset. |
+| **Default Volume Profile** | Checks to make sure you have assigned a **Default Volume Profile Asset** in **Edit** > **Project Settings** > **HDRP Default Settings** that is not the one in the package.<br />It is only required to enable modification on the profile.<br />Press the **Fix** button to copy the one in package into the **Default Resource Folder** and use it instead. |
+| **LookDev Volume Profile** | Checks to make sure you have assigned a **LookDev Volume Profile Asset** in **Edit** > **Project Settings** > **HDRP Default Settings**  that is not the one in the package.<br />It is only required to use the LookDev and want to be able to modify the profile used in it.<br />Press the **Fix** button to copy the one in package into the **Default Resource Folder** and use it instead. |
+
+**Current Quality:**
+
+| **Configuration Option** | **Description**                                              |
+| ------------------------ | ------------------------------------------------------------ |
+| **Assigned - Quality**   | Checks to make sure you have assigned either an [HDRP Asset](HDRP-Asset.md) or null to the **Quality Settings** field corresponding to the current used quality (menu: **Edit** > **Project Settings** > **Quality**).<br />If the value is null, all **Current Quality** related configuration will be the one from the [HDRP Asset](HDRP-Asset.md) used in **Global**.<br />Press the **Fix** button to nullify the field. |
+| **SRP Batcher**          | Checks to make sure that SRP Batcher is enabled.<br />Press the **Fix** button to enable it in the used HDRP Asset. |
 
 <a name="VRTab"></a>
 
 ### HDRP + VR
 
-This tab provides all of the configuration options from the [HDRP tab](#HDRPTab) as well as extra configuration options to help you set your HDRP Project up to support virtual reality. If you can not find an option in this section of the documentation, check the [HDRP tab](#HDRPTab) above. This is only supported on Windows OS.
+This tab provides all of the configuration options from the [HDRP tab](#HDRPTab) as well as extra configuration options to help you set your HDRP Project up to support virtual reality. If you can not find an option in this section of the documentation, check the [HDRP tab](#HDRPTab) above. This is only supported on Windows OS. The extra configuration are all in the **Global** scope.
```

Comment:
Recommended change: 

You can adjust the extra configuration options in the  **Global** scope.